### PR TITLE
Issue #3: Logger adjustment: remove useless user data from the logs.

### PR DESCRIPTION
**Pull request by FGM**

### CODE REVIEW

**Comment by FGM** [comment]
This first version adds a new UserFilterProcessor: this seems to be unnecessary load, better just remove the information in the user processor.

File: lib/Processors/MeteorUserProcessor.js [hunk 1]
```diff
@@ -194,11 +196,18 @@ var MeteorUserProcessor = function (_ProcessorBase) {
   }, {
     key: 'process',
     value: function process(context) {
+      var user = this.getUser();
+
+      // Cannot delete property from undefined or null.
+      if (user && user.services) {
```
Comment (lhamrit) [inline lib/Processors/MeteorUserProcessor.js]: `if (user && user.hasOwnProperty('services')) { ...` ?

File: src/Processors/MeteorUserProcessor.js [hunk 2]
```diff
@@ -135,11 +137,18 @@ export default class MeteorUserProcessor extends ProcessorBase {
 
   /** @inheritdoc */
   process(context) {
+    let user = this.getUser();
+
+    // Cannot delete property from undefined or null.
+    if (user && user.services) {
```
Comment (lhamrit) [inline src/Processors/MeteorUserProcessor.js]: `if (user && user.hasOwnProperty('services')) { ...` ?

**Comment by lhamrit** [comment]
OK :)

**Pull request merged**